validator/sql_public_ip.rego

```diff
@@ -0,0 +1,16 @@
+package templates.gcp.SQLPUBLICIPV1
```

Comment:
Can you add the Google license header?

Also, package should match constraint name; ie. `GCPSQLPublicIpConstraintV1`.